src/main/java/net/minecraftforge/common/ForgeHooks.java

```diff
@@ -220,37 +220,40 @@ public static boolean isCorrectToolForDrops(@Nonnull BlockState state, @Nonnull
         return player.hasCorrectToolForDrops(state);
     }
 
-    public static boolean isCorrectToolForDrops(TieredItem tieredItem, ItemStack stack, BlockState state)
+    public static boolean isCorrectToolForDrops(ItemStack stack, BlockState state)
```

Comment:
I think this should be named `isCorrectTierForDrops` as its just doing tier checks. Gives the modder a clue that this alone is an incomplete check.